prov/efa/src/rdm/rxr_pkt_type_base.c

```diff
@@ -441,10 +453,10 @@ int rxr_pkt_copy_data_to_cuda(struct rxr_ep *ep,
 		 * to achieve best latency.
 		 */
 		if (rxe->bytes_copied + data_size == rxe->total_len) {
-			ofi_copy_to_hmem_iov(desc->peer.iface, desc->peer.device.reserved,
-					     rxe->iov, rxe->iov_count,
-					     data_offset + ep->msg_prefix_size,
-					     data, data_size);
+			ofi_gdrcopy_to_cuda_iov((uint64_t)desc->peer.hmem_data,
+			                        rxe->iov, rxe->iov_count,
+			                        data_offset + ep->msg_prefix_size,
+			                        data, data_size);
 			rxr_pkt_handle_data_copied(ep, pkt_entry, data_size);
```

Comment:
I double checked in my editor - I find it more reliable to achieve "github correct" indent with explicit spaces in the case of multi-line function calls, for my configuration.

Other places, though rendering correctly on github, show up crooked in my editor.

I'm unlikely alone since I see similar indent format choices.